WootricSDK/WootricSDK/WTRSurveyViewController.m

```diff
@@ -173,12 +200,11 @@ - (void)twitterButtonPressed {
     [tweetSheet setInitialText:[NSString stringWithFormat:@"%@ @%@", [_feedbackView feedbackText], _settings.twitterHandler]];
```

Comment:
the code of ```twitterButtonPressed``` and ```facebookButtonPressed``` is almost the same, you can move the similar code to a primate method and reduce duplicity :)